python/hsfs/core/vector_server.py

```diff
@@ -366,19 +368,6 @@ def _apply_transformation(self, row_dict: Dict[str, Any]) -> Dict[str, Any]:
                 row_dict[feature_name] = transformation_fn(row_dict[feature_name])
         return row_dict
 
-    @property
```

Comment:
One more thing that I noticed is that `training_dataset.py` is also using the required_serving_keys from the vector sever in the function `serving_keys` function. But since `TrainingDataset` is Deprecated it should be fine I think.